crates/lox_core/src/time/continuous/tt.rs

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2024. Helge Eichhorn and the LOX contributors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+//! Module tt provides a type representing instants in the TT timescale and functions for
+//! converting between TT and other timescales.
+
+use crate::time::continuous::{RawTime, TimeDelta, UT1};
+use crate::time::WallClock;
+use crate::wall_clock;
+
+/// Terrestrial Time. Defaults to the J2000 epoch.
+#[derive(Debug, Default, Copy, Clone, Eq, PartialEq)]
+pub struct TT(RawTime);
+
+wall_clock!(TT, tt_wall_clock_tests);
+
+impl TT {
+    pub fn new(t: RawTime) -> Self {
+        Self(t)
+    }
+
+    /// Convert from TT to UT1 given a user-supplied value for ΔT.
+    pub fn to_ut1(&self, dt: TimeDelta) -> UT1 {
```

Comment:
We will need to be more sophisticated here. UT1 and by extension all offsets to other time scales are measured quantities and need to be determined via IERS tables for example.
I would propose to add a `UT1Provider` trait (or something like that) and pass an `impl UT1Provider` argument to the conversions from and to UT1.

Reply:
Yes, ERFA does not handle measured quantities but we should. Although we should also enable the user to provide their own values.